examples/apps/presence-tracker/src/Audience.ts

```diff
@@ -0,0 +1,32 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { IClient } from "@fluidframework/protocol-definitions";
+import { ServiceAudience } from "fluid-framework";
+import {
+	ITinyliciousAudience,
+	TinyliciousMember,
+	TinyliciousUser,
+} from "@fluidframework/tinylicious-client";
+import { assert } from "@fluidframework/common-utils";
+
+export class MockAudience
+	extends ServiceAudience<TinyliciousMember>
+	implements ITinyliciousAudience
+{
+	protected createServiceMember(audienceMember: IClient): TinyliciousMember {
+		const tinyliciousUser = audienceMember.user as TinyliciousUser;
+		assert(
+			tinyliciousUser !== undefined && typeof tinyliciousUser.name === "string",
+			0x313 /* Specified user was not of type "TinyliciousUser". */,
```

Comment:
This probably blows up policy check due to duplicate assert code.  Since this is just for an example, probably change this assert to `throw` an Error instead.

Reply:
Also does this work for local-driver?  I didn't realize it put a .name on the user too.

Reply:
The test `Current User is displayed` seems to give a timeout error.

Reply:
Does the example appear to work if you run it via `start:client:test`?  If not, that's usually the easiest place to debug.

Reply:
Yep, it runs via `start:client:test`

Reply:
test cases pass :)